xarray/core/dataset.py

```diff
@@ -2384,6 +2384,34 @@ def from_dataframe(cls, dataframe):
             obj[name] = (dims, data)
         return obj
 
+    def to_dask_dataframe(self, set_index=True):
```

Comment:
Of course, that makes sense.  set_index=False is the default (index is simply an integer as would be from calling df.reset_index() )